packages/fxa-auth-server/test/scripts/verification-reminders.js

```diff
@@ -0,0 +1,21 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const ROOT_DIR = '../..';
+
+const cp = require('child_process');
+const Promise = require(`${ROOT_DIR}/lib/promise`);
+const path = require('path');
+
+cp.execAsync = Promise.promisify(cp.exec);
```

Comment:
Could you just use [execSync](https://nodejs.org/docs/latest-v10.x/api/child_process.html#child_process_child_process_execsync_command_options), depending on this behavior:

> If the process times out or has a non-zero exit code, this method will throw. The Error object will contain the entire result from child_process.spawnSync().